t/08-autotest.t

```diff
@@ -450,7 +450,6 @@ subtest make_snapshot => sub {
 };
 
 subtest loadtestdir => sub {
-    use lib 't/data/tests';
     $bmwqemu::vars{CASEDIR} = 't/data/tests';
     stderr_like {
         like warning {
```

Comment:
Do you know what this test does?
It calls the `warning` function from Test::Warnings, which returns an arrayref if there is more than one warning, and if you compare that to a string, it will produce `ARRAY(0x...)`, which the person writing this test thought is a good check.
But it only checks if there is a warning, not even the content of the warning.

And the ticket says that the new code should not produce a warning anymore. You should modify the test accordingly.
Just run it without the `like warning { }` and see what output you get.